apps/backend/src/schemas/proxy_config.py

```
@@ -0,0 +1,225 @@
+"""
+Proxy Configuration Schemas
+
+Pydantic models for SWAG reverse proxy configuration management
+including file-based nginx configurations and database tracking.
+"""
+
+from datetime import datetime
+from typing import Dict, List, Optional, Any, Union
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)